Gemfile.lock

```diff
@@ -327,4 +330,4 @@ DEPENDENCIES
   validates_email_format_of
 
 BUNDLED WITH
-   1.12.5
+   1.14.3
```

Comment:
Just ran `docker-compose run web bundle install` for this. That field just marks the previously bundled with version -- it doesn't lock future bundles into using that version.